adafruit_il0373.py

```diff
@@ -31,6 +31,12 @@
 
 import displayio
 
+try:
+    from typing import Optional
+    from displayio import Fourwire
```

Comment:
Looks like a typo of `FourWire`, don't forget to change it in `__init__()` as well!